Create polkabox.md

irene-bbox opened the pull request:
### Project Abstract

>PolkaBox is an open data platform that enriches blockchain data by (1) visualizing Polkadot on-chain data, (2) interpreting it relative to external factors (off-chain data from public APIs), and (3) generating predictions around key token metrics from the aggregation of the two. The platform offers a transparent analysis of the parachain transactions generated in the Polkadot ecosystem. Concurrently, our platform fetches real-world off-chain data (e.g., news feeds, election campaigns, Reddit, Tweets, crypto media, weather, COVID-19, WHO, etc.) and leverages Machine Learning models to gauge the market sentiment on a given chain or asset. PolkaBox seeks to be the first robust aggregator of on-chain and off-chain data for Polkadot. Finally, the goal is to publish developer-friendly APIs for anyone to build on top of PolkaBox using raw and/or enriched data to tackle real-world problems.


### For which grant [level](https://github.com/w3f/Grants-Program#level_slider-levels) are you applying? 
- [ ] **Level 1**:  Up to $10,000, 2 approvals
- [x] **Level 2**:  Up to $50,000, 3 approvals
- [ ] **Level 3**:  Unlimited, 5 approvals (for > $100k Web3 Foundation Council approval)

### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied, renamed ( `project_name.md`) and updated.
- [x] A BTC or Ethereum (DAI/USDT) address for the payment of the milestones is provided inside the application.
- [x] I have read and acknowledged the [terms and conditions](https://github.com/w3f/Grants-Program/blob/master/docs/T&Cs.md).
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process).

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/w3f/Grants-Program?pullRequest=717) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/w3f/Grants-Program?pullRequest=717) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/w3f/Grants-Program?pullRequest=717) it.</sub>

semuelle commented:
Thanks for the PR + email, Irene. I will continue reviewing the application and provide some feedback in the coming days. If it's okay with you, I would continue the conversation here.

As the commit author, could @MichaelBrink sign the [CLA](https://github.com/w3f/Grants-Program/pull/717#issuecomment-982892723), please?

MichaelBrink commented:
Hi Sebastian sure thing!

MichaelBrink commented:
CLA signed @semuelle :)

Noc2 reviewed (changes requested):
Thanks for the application. Something you should be aware of is that our grants program focuses on the development of open source software. The deliveries of your milestone 1 seem to focus on hosting a node and some endpoints. Do you also plan to open source the software, which you need to develop for this or are you going to use existing tools/indexer? How will your solution be different from other existing solutions, see https://github.com/w3f/Grants-Program/blob/master/docs/polkadot_stack.md Feel to also directly update the deliverieables. The more technical details, the better. These are basically the requirements of our contracts.

irene-bbox commented:
Thank you for your feedback @Noc2  ! We agree with all of your comments, which we implemented in our updated submission (see master GitHub repo here: https://github.com/BalloonBox-Inc/polkadot-analytics). Specifically, here are our responses to your feedback:

1. Milestone 1 initially focused on a hosted node and endpoints. After having a team briefing we realized it is actually best to deploy existing tools/indexers instead to extract on-chain data. This has multiple advantages: it's a guarantee for transparency of open-source; we can deploy Polkadot-specific indexing solutions like SubQuery; GraphQL is not only a fast and performant data query language, but our team is well familiar with the GraphQL syntax, and so we're keen on deploying it to fetch blockchain data. 

2. PolkaBox is set apart from other projects because it is the first platform with a strong focus on prediction coupled with Market Sentiment Analysis; it is the first platform aggregating news feed for the Polkadot ecosystem, it is the first platform built on Python as core language, and it offers Python SDKs for any developer to run our predictive and sentiment models in their local machine. 

3. We added specifications to our Milestones deliverables. For a full overview of them, please, consult the 'Development Roadmap' section of our application. 

Thank you again for your feedback. We hope our changes help you gain a tangible idea for the PolkaBox prototype.

Noc2 reviewed (changes requested):
Thanks for the update. However, I have to be honest with you. I’m not sure this application is a good fit for our grants program. We usually focus on open source projects and technology that potentially can be used by others. Your milestone 1 still seems to focus on hosting/deployment. Also we usually don’t like to focus on specific tokens or token price, so things like Sentiment Analysis isn’t really a good fit for our grants program either. It might make more sense for you to apply for treasury funding. If you are interested in this, I could put you in touch with Raul.

irene-bbox commented:
Thank you @Noc2 for your fast response! We're completely aligned to open-source everything, but it seems as per your comments that the use case is not aligned. We are just trying to be pragmatic about software access/ownership like databases but anyone could spin up their own databases or use PolkaBox's public APIs (for external data mining and storage). We're looking into treasury funding right now and will deliberate by COB today on the next steps. Meanwhile, we thank you for your support thus far!

Noc2 reviewed (changes requested):
Thanks for the reply. I have to disagree with your treasury assessment. The treasury officially has no restrictions, so I don’t believe there is any project that can not be theoretically funded via the treasury. It’s always up to the on-chain council. Also the treasury is highly transparent, since everything is on-chain and you need to involve the community. 
Compared to the treasury, the grants program has more restrictions. One of them is that we usually don’t cover hosting or deployment costs via the grants program. For example, we won’t support the following deliveries of your milestone 1 “Indexer Instances” or hosting of endpoints. Btw. I believe the treasury funded public endpoints before. We would only support the open source development work of unique features for these endpoints. Additionally, as I said before, we usually don’t care too much about the token price as web3 foundation, which is why I think the entire application will be hard to approve. In any case, I will share the application with the rest of the team, but I won’t approve it because of the above reasons.

irene-bbox commented:
Thank you for your explanation regarding the scope of the Treasury Funding. We may reconsider it, yet we're grateful we can proceed with our application through W3F as well. We understand and respect your decision, and we appreciate you shared the application with the team. We look forward to hearing back!

semuelle reviewed (commented):
Hi Irene. I have re-read your application, and David already pointed out the issues I am seeing. The price of tokens is not something we want to draw even more attention to. And the specifications of your deliverables still contain language that points to building a single platform ("Set up ...", "Deploy ..."). 

I think making data and information more accessible is absolutely worthwhile. This application is just very focused on token prices and market sentiment while there is so much other interesting data to gather. I would also be much more inclined to support a grant focused on shipping CLI tools and documentation/tutorials on how to run analyses locally.

MichaelBrink commented:
@semuelle - makes complete sense! I appreciate the focus on a suite of dev tools for self analysis. Perhaps this is something we can look into next year. Currently we're trying to play to our strengths which is a strong data science/ML component to the team to spruce up software development :) I know the Web3 grants panel is super busy so personally, thanks so much for actually taking the time to evaluate & debrief - much appreciated and Merry Christmas!

alxs commented:
Hi @irene-bbox @MichaelBrink, thanks a lot for your application. It is clear a lot of work went into planning and cleanly presenting the project. However, I'm afraid I share David's concerns to a T and hence also won't approve the application. I would also say that most of the project has little to do with web3 other than perhaps the integration with SubQuery, which is not something we would fund in the first place, as David also pointed out. I'm sorry I don't have better news to share. I hope you can find another avenue to realise the project and wish you a happy holiday season, too.

Noc2 commented:
Since it seems the rest of the committee is also not interested in this application, I’m going to close the application. But are you already in contact with Raul from parity regarding a potential treasury proposal? If not I could introduce you to him via email.

irene-bbox commented:
Dear David,

Thank you for thoroughly considering our application and thank you for
patiently guiding us throughout the revision process.
We understand and agree that the data science focus of our proposed
PolkaBox platform is only partially in line with the dev-heavy scope of the
Web3Foundation.

We appreciate you offering to connect us with Raul from the treasury fund.
As the new year started, we've decided to allocate much of our Data Science
and Software Dev capacity to pursue another project.
We hope for a different and new opportunity to collaborate with Web3 in the
future, though!

Thank you again for your consideration of PolkaBox.
Best wishes to you and the Web3 Team in the new year.

Warm regards,

Irene Fabris

> Message ID: ***@***.***
> com>
>

irene-bbox commented:
Dear David,

Thank you for thoroughly considering our application and thank you for
patiently guiding us throughout the revision process.
We understand and agree that the data science focus of our proposed
PolkaBox platform is only partially in line with the dev-heavy scope of the
Web3Foundation.

We appreciate you offering to connect us with Raul from the treasury fund.
As the new year started, we've decided to allocate much of our Data Science
and Software Dev capacity to pursue another project.
We hope for a different and new opportunity to collaborate with Web3 in the
future, though!

Thank you again for your consideration of PolkaBox.
Best wishes to you and the Web3 Team in the new year.

Warm regards,

Irene Fabris
on behalf of the entire BallloonBox Team



On Wed, Jan 5, 2022 at 5:00 AM David Hawig ***@***.***> wrote:

> Since it seems the rest of the committee is also not interested in this
> application, I’m going to close the application. But are you already in
> contact with Raul from parity regarding a potential treasury proposal? If
> not I could introduce you to him via email.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/w3f/Grants-Program/pull/717#issuecomment-1005665268>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWWVNSAXEWUU5WLGD5G37BLUUQ6GZANCNFSM5JCPIVCQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
>  or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>
> .
> You are receiving this because you were mentioned.
>



On Wed, Jan 5, 2022 at 5:00 AM David Hawig ***@***.***> wrote:

> Closed #717 <https://github.com/w3f/Grants-Program/pull/717>.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/w3f/Grants-Program/pull/717#event-5846282635>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWWVNSBGMS47DBNZ5AR5S6TUUQ6G3ANCNFSM5JCPIVCQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
>  or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>
> .
>
You are receiving this because you were mentioned.
